src/tests/cli_tests.py

```diff
@@ -3715,7 +3719,35 @@ def test_sym_encryption__gpg_to_rnp(self):
     def test_sym_encryption__rnp_to_gpg(self):
         # Encrypt cleartext with RNP and decrypt with GPG
         for size, cipher, z in zip(Encryption.SIZES_R, Encryption.CIPHERS_R, Encryption.Z_R):
-            rnp_sym_encryption_rnp_to_gpg(size, cipher, z)
+            rnp_sym_encryption_rnp_to_gpg(size, cipher, z, 1024)
+
+    def test_sym_encryption_s2k_msec(self):
+        rnp_sym_encryption_rnp_to_gpg(Encryption.SIZES[0], None, Encryption.Z[0], False, 1)
```

Comment:
We should also check (via `--list-packets`) whether encryption used specified iterations value instead of default.

Reply:
yeah, but 
> s2k iterations: 41943040 (encoded as 244)

it does not necessarily match the value passed on command line
I guess the test should use the same formula to check results?
```
The count is coded into a one-octet number using the following formula:

  #define EXPBIAS 6
      count = ((Int32)16 + (c & 15)) << ((c >> 4) + EXPBIAS);

The above formula is in C, where "Int32" is a type for a 32-bit integer, and the variable "c" is the coded count, Octet 10.
```

Reply:
@antonsviridenko just 2-3 sample runs would be enough, the goal of this test would be to make sure that passed value correctly goes from CLI down to all layers. I believe we have more detailed tests, checking this formula, on FFI layer.

Reply:
ok, used `src/tests/s2k-iterations.cpp` as a reference